testing/trino-testing/src/main/java/io/trino/testing/BaseOrcWithBloomFiltersTest.java

```diff
@@ -11,31 +11,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.trino.plugin.hive.orc;
+package io.trino.testing;
 
 import io.trino.Session;
-import io.trino.plugin.hive.HiveQueryRunner;
-import io.trino.testing.AbstractTestQueryFramework;
-import io.trino.testing.QueryRunner;
 import org.intellij.lang.annotations.Language;
 import org.testng.annotations.Test;
 
 import static io.trino.testing.sql.TestTable.randomTableSuffix;
 import static java.lang.String.format;
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class TestOrcWithBloomFilters
+public abstract class BaseOrcWithBloomFiltersTest
```

Comment:
Can we move this extraction part to a different commit.

Reply:
Are functional and test cases submitted separately?

Reply:
No, it can just be nice to have refactoring commits separate. Separating commits where the code should be identical, but in a new file from commits where the code is changed

Reply:
I have submitted a new pr #12183.

Reply:
I guess this can be the first commit with a message `Introduce BaseOrcWithBloomFiltersTest` and `TestHiveOrcWithBloomFilters` can be a part of this commit. This test class `TestIcebergOrcWithBloomFilters` can be a part of Iceberg connector support ORC bloom filter commit

Reply:
Done.